tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -25,13 +25,13 @@
 
 void ROOT::Experimental::RNTupleModel::EnsureValidFieldName(std::string_view fieldName)
 {
+   RResult<void> nameValid = Detail::RFieldBase::EnsureValidFieldName(fieldName);
```

Comment:
Not for this PR, but maybe it makes sense to add a shorthand to `RResult<void>` so that we can write

    Detail::RFieldBase::EnsureValidFieldName(fieldName).ThrowOnError();

Reply:
Good idea, I opened #5975 for this :)